internal/unit_tests/helm_template_primaries_test.go

```diff
@@ -598,6 +599,88 @@ func TestDuplicateImageCredentials(t *testing.T) {
 	}))
 }
 
+//TestMissingApocConfigMap ensure the apoc config map is not created when apoc configs are not provided in values.yaml
+func TestMissingApocConfigMap(t *testing.T) {
+	t.Parallel()
+
+	forEachPrimaryChart(t, andEachSupportedEdition(func(t *testing.T, chart model.Neo4jHelmChart, edition string) {
+		manifest, err := model.HelmTemplate(t, chart, useDataModeAndAcceptLicense)
+		if !assert.NoError(t, err) {
+			return
+		}
+		configMaps := manifest.OfType(&v1.ConfigMap{})
+		if !assert.NotEqual(t, len(configMaps), 0) {
+			return
+		}
+		var apocConfigMapPresent bool
+		for _, configMap := range configMaps {
+			if strings.Contains(configMap.(*v1.ConfigMap).Name, "apoc-config") {
+				apocConfigMapPresent = true
+				break
+			}
+		}
+		//apoc-config configMap should not be present since we are not passing apoc configs
+		if !assert.Equal(t, apocConfigMapPresent, false, "apocConfigMap present !!! Should NOT BE present infact") {
+			return
+		}
+	}))
+}
+
+//TestMissingApocConfigMap ensure the apoc config map is not created when apoc configs are not provided in values.yaml
```

Comment:
The comment here is the same as `TestMissingApocConfigMap` function comment, I guess this is just a copy/paste typo.

Reply:
fixed